python/hail/methods/impex.py

```diff
@@ -615,16 +615,122 @@ def import_table(paths, key=[], min_partitions=None, impute=False, no_header=Fal
            reference_genome=nullable(GenomeReference),
            contig_recoding=nullable(dictof(strlike, strlike)),
            drop_chr0=bool)
-def import_plink(bed, bim, fam, min_partitions=None, delimiter='\\\\s+',
-                 missing='NA', quant_pheno=False, a2_reference=True, reference_genome=None,
-                 contig_recoding={'23': 'X', '24': 'Y', '25': 'X', '26': 'MT'}, drop_chr0=False):
+def import_plink(bed, bim, fam,
+                 min_partitions=None,
+                 delimiter='\\\\s+',
+                 missing='NA',
+                 quant_pheno=False,
+                 a2_reference=True,
+                 reference_genome=None,
+                 contig_recoding={'23': 'X',
+                                  '24': 'Y',
+                                  '25': 'X',
+                                  '26': 'MT'},
+                 drop_chr0=False):
+    """Import PLINK binary file (BED, BIM, FAM) as a :class:`.MatrixTable`.
+
+    Examples
+    --------
+
+    Import data from a PLINK binary file:
+
+    >>> ds = methods.import_plink(bed="data/test.bed",
+    ...                            bim="data/test.bim",
+    ...                            fam="data/test.fam")
+
+    Notes
+    -----
+
+    Only binary SNP-major mode files can be read into Hail. To convert your
+    file from individual-major mode to SNP-major mode, use PLINK to read in
+    your fileset and use the ``--make-bed`` option.
+
+    The centiMorgan position is not currently used in Hail (Column 3 in BIM
+    file).
+
+    The ID (``s``) used by Hail is the individual ID (column 2 in FAM file).
+
+    .. warning::
+
+        No duplicate individual IDs are allowed.
+
+    The produced :class:`.MatrixTable` has one column-indexd field:
```

Comment:
You also need to include v as row field, s as col field, and the entry field(s). I think it's clearer to organize this all as:

---

The resulting matrix table has the following fields.

Column fields:

Row fields:

Entry field: